renv.lock

```diff
@@ -1,655 +1,501 @@
+			"Version" : "2.1.1",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"Andromeda" : {
+			"Package" : "Andromeda",
+			"Version" : "0.4.1",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"DatabaseConnector" : {
+			"Package" : "DatabaseConnector",
+			"Version" : "4.0.2",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"ggplot2" : {
+			"Package" : "ggplot2",
+			"Version" : "3.3.3",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"gridExtra" : {
+			"Package" : "gridExtra",
+			"Version" : "2.3",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"ParallelLogger" : {
+			"Package" : "ParallelLogger",
+			"Version" : "2.0.1",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"plyr" : {
+			"Package" : "plyr",
+			"Version" : "1.8.6",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"RcppEigen" : {
+			"Package" : "RcppEigen",
+			"Version" : "0.3.3.9.1",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"readr" : {
+			"Package" : "readr",
+			"Version" : "1.4.0",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"survival" : {
+			"Package" : "survival",
+			"Version" : "3.2-11",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"Cyclops" : {
+			"Package" : "Cyclops",
+			"Version" : "3.1.1",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"EmpiricalCalibration" : {
+			"Package" : "EmpiricalCalibration",
+			"Version" : "2.1.0",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"pROC" : {
+			"Package" : "pROC",
+			"Version" : "1.17.0.1",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"renv" : {
+			"Package" : "renv",
+			"Version" : "0.13.2",
+			"Source" : "Repository",
+			"Repository" : "CRAN"
+		},
+		"FeatureExtraction" : {
+			"Package" : "FeatureExtraction",
+			"Version" : "3.1.1",
+			"Source" : "GitHub",
+			"RemoteType" : "github",
+			"RemoteHost" : "api.github.com",
+			"RemoteRepo" : "FeatureExtraction",
+			"RemoteUsername" : "ohdsi",
+			"RemoteRef" : "v3.1.1"
+		},
+		"CohortMethod" : {
+			"Package" : "CohortMethod",
+			"Version" : "4.1.0",
+			"Source" : "GitHub",
+			"RemoteType" : "github",
+			"RemoteHost" : "api.github.com",
+			"RemoteRepo" : "CohortMethod",
+			"RemoteUsername" : "ohdsi",
+			"RemoteRef" : "v4.1.0"
+		},
+		"MethodEvaluation" : {
+			"Package" : "MethodEvaluation",
+			"Version" : "2.1.0",
+			"Source" : "GitHub",
+			"RemoteType" : "github",
+			"RemoteHost" : "api.github.com",
+			"RemoteRepo" : "MethodEvaluation",
+			"RemoteUsername" : "ohdsi",
+			"RemoteRef" : "v2.1.0"
+		},
+		"OhdsiSharing" : {
+			"Package" : "OhdsiSharing",
+			"Version" : "0.2.2",
+			"Source" : "GitHub",
+			"RemoteType" : "github",
+			"RemoteHost" : "api.github.com",
+			"RemoteRepo" : "OhdsiSharing",
+			"RemoteUsername" : "ohdsi",
+			"RemoteRef" : "v0.2.2"
+		},
+		"SkeletonComparativeEffectStudy" : {
```

Comment:
Could you remove the actual skeleton itself from the lock file? You can do this by either just removing this entry manually from the lock file, or by setting the new `includeRootPackage` argument to FALSE in the newly updated [OhdsiRTools](http://ohdsi.github.io/OhdsiRTools/reference/createRenvLockFile.html).